.github/workflows/create-rc-tag.yml

```diff
@@ -0,0 +1,97 @@
+name: Create RC Tag
+# Required secrets for this workflow: MAINTAIN_USERNAME, MAINTAIN_EMAIL, SLACK_TOKEN, MAINTAIN_TOKEN, and a var: SLACK_TAG_CHANNELS.
+
+env:
+  CHANGELOG_FILE: 'temp-changelog.txt'
+
+on:
+  workflow_dispatch:
+    inputs:
+      channel:
+        type: choice
+        description: 'The channel to get the version to'
+        required: true
+        default: 'cloud'
+        options:
+          - 'ga'
+          - 'cloud'
+          - 'beta'
+jobs:
+  create-tag:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.MAINTAIN_TOKEN }}
+      - name: Get Configuration
+        id: configuration
+        uses: maork-elementor/json-to-variables@main
+        with:
+            filename: './.github/workflows/config.json'
+            prefix: config
+      - name: Test permissions
+        uses: ./.github/workflows/permissions
+        with:
+          ENVIRONMENT: ${{ env.config_deployment_environment }}
+          DEPLOYMENT_PERMITTED: ${{ env.config_deployment_permitted }}
+          DEPLOYMENT_REPOSITORY_OWNER: ${{ env.config_deployment_repository_owner }}
+      - name: Install Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+      - name: Env Vars Configure
+        uses: ./.github/workflows/env-vars-configure
+        with:
+          CHANNEL: ${{ github.event.inputs.channel }}
+      - name: Bump Channel Version
+        uses: ./.github/workflows/bump-channel-version
+        with:
+          CLEAN_PACKAGE_VERSION: ${{ env.PACKAGE_VERSION_CLEAN }}
+          CHANNEL: ${{ github.event.inputs.channel }}
+      - name: Install Dependencies
+        run: |
+          npm ci
+      - name: Build plugin
+        uses: ./.github/workflows/build-plugin
+        with:
+          PACKAGE_VERSION: ${{ env.PACKAGE_VERSION }}
+          BUILD_SCRIPT_PATH: "./.github/scripts/build-zip.sh"
+          UPDATE_VERSIONS_SCRIPT_PATH: ""
```

Comment:
Remove this and make the empty value a default one

Reply:
Done